relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -688,6 +688,7 @@ pub fn extract_measurements(span: &mut Span, is_mobile: bool) {
                     &data.messaging_message_body_size,
                     "messaging.message.body.size",
                 ),
+                (&data.messaging_message_id, "messaging.message.id"),
```

Comment:
This will try to convert `data.messaging_message_id` to a floating point number and insert it as `measurement.id`, which will always fail.